### PR TITLE
Fix ghost "s email server" translation

### PR DESCRIPTION
**Pull request by dannyroberts**
## Technical Summary

Fix ghost "s email server" translation stemming from an unsupported string style in web_users.js. I believe the backtick (`` ` ``) character isn't getting interpreted as bounding a string, and instead it's treating the `'` in the word `user's` as the start of a string. By rearranging the syntax used here to something more vanilla, I think we'll not only correctly translate this string, but also prevent the instability around whether this `s email servers` string gets collected at all—sometimes it was getting collected and sometimes it wasn't, causing weird diffs back and forth.

I also snuck in a very minor change to `make-translations.sh` that makes the post-checkout hook not run on the git hack it does towards the end to get rid of some irrelevant changes. This makes it go a bit smoother since it's doing `checkout` in a loop over all the changed locale files.

### Safety story
This is a pretty clear change to one string on one page, so it's pretty easy to test manually that it still works. I tested it and (1) the page doesn't have a JS error or anything (2) the string shows up just the way it's intended. Screenshot:

Screen Shot 2022-06-06 at 5 35 50 PM

### Automated test coverage

I think the relevant tests for my change are the ones that test that running `make translations` again does not create any additional diff.

### QA Plan
No formal QA or further testing planned.

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/users/static/users/js/web_users.js
Comment (dannyroberts) [inline corehq/apps/users/static/users/js/web_users.js:213]:
```suggestion
                fromAddress: initialPageData.get('fromAddress'),
```

File: corehq/apps/users/static/users/js/web_users.js
Comment (dannyroberts) [inline corehq/apps/users/static/users/js/web_users.js:214]:
```suggestion
            }),
```

**Comment by dannyroberts** [comment]
@akashkj if you are waiting on this, once you or someone else approve it, feel free to merge it. (I'm comfortable with the change and don't need to time the merge carefully or anything.)

**Review by akashkj — APPROVED** [review]
Thanks @dannyroberts

**Pull request merged**
